sklearn/feature_selection/_from_model.py

```diff
@@ -341,14 +341,29 @@ def fit(self, X, y=None, **fit_params):
             classification, real numbers in regression).
 
         **fit_params : dict
-            Other estimator specific parameters.
+            - If `enable_metadata_routing=False` (default):
```

Comment:
I think that it will not be obvious yet for people what means `enable_metadata_routing=False/True`. We need to be more explicit by stating that the option is set in `set_config` or with `config_context`. Doing so, we will have an automatic sphinx reference where a user can be redirected and understand more about the metadata routing business.

Reply:
I appreciate your suggestion, and it's important to help the users understand what metadata routing is. In this case, we already have a sphinx reference a few lines below this code snippet  and a link to the user guide:

```
                .. versionchanged:: 1.4
                    `**fit_params` are routed to the sub-estimator, if
                    `enable_metadata_routing=True` is set via
                    :func:`~sklearn.set_config`, which allows for aliasing.

                See :ref:`Metadata Routing User Guide <metadata_routing>` for
                more details.
```

I believe that adding more information might become overwhelming. However, if you strongly feel that additional clarification is necessary, I will add it. What do you think, @glemaitre ?

Reply:
Indeed, it might be enough. Could we just mention the `config_context` together with the `set_config`.

Reply:
Yes, I will add that.